src/test/java/com/salesforce/dynamodbv2/UpdateTest.java

```diff
@@ -69,6 +79,30 @@ void update(TestArgument testArgument) {
         });
     }
 
+    @ParameterizedTest
+    @ArgumentsSource(DefaultArgumentProvider.class)
+    void updatePrimaryKeyFieldFails(TestArgument testArgument) {
```

Comment:
New test case: can't refer to table PK fields in an update expression and throw exception consistent with single-tenant dynamo (though I got the error message from local dynamo and I'm not sure if that's necessarily the same as what AWS dynamo would give?)